### PR TITLE
Service catalog

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

File: chapter_4/service_catalogs_and_user_self-service_portal.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Introduction to ManageIQ services
 
- Services are one of the most interesting feature within ManageIQ. Services can include one or more virtual machines, instances etc. that can be deployed across hybrid environments.
+ Services are one of the most interesting feature within ManageIQ. Services can include one or more virtual machines, instances, etc. that can be deployed across hybrid environments.
 
 ### User self-service portal
```
Comment (KhushbuB) [inline chapter_4/service_catalogs_and_user_self-service_portal.md]: Please make the link click able.

Comment (son-vyas) [reply chapter_4/service_catalogs_and_user_self-service_portal.md]: @KhushbuB , If I am not wrong then then you mean this link -> 
" https:// your_manageiq_appliance / self_service/ " .
 This link is actually not supposed to be clickable.
 The i.p address of user's ManageIQ appliance has to be replaced with <your_manageiq_appliance>.

Comment (KhushbuB) [reply chapter_4/service_catalogs_and_user_self-service_portal.md]: Oops i got it. Sorry :(

Comment (son-vyas) [reply chapter_4/service_catalogs_and_user_self-service_portal.md]: It's alright :)

**Pull request merged**